source/features/repo-wide-file-finder.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'dom-chef';
+import features from '../libs/features';
+import select from 'select-dom';
+import {getRepoURL} from '../libs/utils';
+import getDefaultBranch from '../libs/get-default-branch';
+
+async function init(): Promise<void> {
+	if (!select.exists('[data-hotkey="t"]')) {
+    select('#js-repo-pjax-container')?.appendChild(
+	<a
+		hidden
+		data-hotkey="t"
+		data-pjax="true"
+		href={`/${getRepoURL()}/find/${await getDefaultBranch()}`}
+	/>
+		);
```

Comment:
Also ensure that the indentation is correct everywhere. Normally the linter should fix that but I don’t know why it’s not. I’m seeing it misaligned it here

Reply:
This happened after running `npm run lint-fix`. I thought it was some issue with my editor. Don't know how to fix this as whatever I do, linting puts it back at this.